enforcer-rules/src/test/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRuleTest.java

```diff
@@ -101,9 +102,15 @@ private void setupMock()
         .thenReturn(mockDependencyResolutionResult);
     when(mockRuleHelper.evaluate("${session}")).thenReturn(mockMavenSession);
     when(mockRuleHelper.evaluate("${project}")).thenReturn(mockProject);
+    when(mockProject.getArtifact()).thenReturn(new org.apache.maven.artifact.DefaultArtifact(
+        "com.google.cloud", "linakge-checker-rule-test", "0.0.1", "compile",
```

Comment:
linkage

Reply:
Fixed.

Reply:
doesn't look like it is. still spelled linakge

Reply:
(My bad, I fixed another place) Updated now.